common/include/PS2Edefs.h

```diff
@@ -564,6 +564,8 @@ typedef void (CALLBACK* _PS2EsetEmuVersion)(const char* emuId, u32 version);		//
 // GS
 // NOTE: GSreadFIFOX/GSwriteCSR functions CANNOT use XMM/MMX regs
 // If you want to use them, need to save and restore current ones
+typedef void (CALLBACK* _GSosdLog)(const char *utf8, u32 color);
```

Comment:
If they took the same arguments I might agree, but one takes a single string for display in the log and the other takes two strings one for key and one for value.  I think they should remain separate instead of overloading it to do multiple different things.